<launch>
    <node pkg="netft_rdt_driver" name="ft7" type="netft_node" args="192.168.0.123" respawn="true">
        <remap from='netft_data' to='/force_torque_ft7' />
    </node>

    <node name="static_tf_ft7_broadcaster" pkg="tf"
        type="static_transform_publisher"
        args="0 0 0 0 0 0 1 world force_torque_ft7 50"
        respawn="false" output="screen"/>
</launch>

